system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_endpoint_test.clj

```diff
@@ -119,11 +126,20 @@
 
             "Invalid update field"
             {:concept-ids ["C1", "C2", "C3"]
+             :name "TEST NAME"
              :update-field "Science keywords"
              :update-type "ADD_TO_EXISTING"
              :update-value {:Category "EARTH SCIENCE"}}
             400
-            ["/update-field instance value (\"Science keywords\") not found in enum (possible values: [\"SCIENCE_KEYWORDS\",\"LOCATION_KEYWORDS\",\"DATA_CENTERS\",\"PLATFORMS\",\"INSTRUMENTS\"])"]))))
+            ["/update-field instance value (\"Science keywords\") not found in enum (possible values: [\"SCIENCE_KEYWORDS\",\"LOCATION_KEYWORDS\",\"DATA_CENTERS\",\"PLATFORMS\",\"INSTRUMENTS\"])"]
+
+            "Missing name value"
```

Comment:
Change this to check the value for name defaults to the task ID instead of causing an error.